JHipster Conf 2 (#124)
ippontech/blog-usa: posts/JHipster-Conf-2-Hottest-JHipster-Day-of-the-Year.md

---
authors:
- Theo Lebrun
tags:
- JHipster
date: 2019-07-01T17:27:07.000Z
title: "JHipster Conf 2 - Hottest JHipster Day of the Year"
image: https://github.com/falydoor/blog-usa/blob/jhipster-conf-2/images/2019/07/jhipster-conf.png
---

June 27 was one of the hottest days (90°F) in beautiful Paris, but it was also the hottest JHipster day of the year with its worldwide event: the JHipster Conf 2! 300 people attended (almost twice last year's number) from all over the world with 19 different countries including Australia and South Africa. With English and French tracks, the conference covered many different topics with speakers from the JHipster core team and from big organizations such as Google or Pivotal.

> This year was also different since we were happy to welcome speakers with real-life stories which make the beauty of the Open Source community even bigger. [Pia Mancini](https://twitter.com/piamancini) who is the cofounder of [Open Collective](https://opencollective.com/) gave an awesome talk on how Open Collective helps Open Source communities by being financially transparent. [Cindy Marin](https://twitter.com/laslorma) and [Ernesto Butto](https://twitter.com/poolebu) shared their really touching story on how JHipster started a positive chain of events, you can read more about it on [Medium](https://medium.com/@ernesto_31246/a-thank-you-letter-to-the-jhipster-team-7397ad8a09fb).

Reviewer: ```suggestion
This year was also different since we were happy to welcome speakers with real-life stories which make the beauty of the Open Source community even better. [Pia Mancini](https://twitter.com/piamancini), who is the cofounder of [Open Collective](https://opencollective.com/), gave an awesome talk on how Open Collective helps Open Source communities by being financially transparent. [Cindy Marin](https://twitter.com/laslorma) and [Ernesto Butto](https://twitter.com/poolebu) shared their really touching story on how JHipster started a positive chain of events, you can read more about it on [Medium](https://medium.com/@ernesto_31246/a-thank-you-letter-to-the-jhipster-team-7397ad8a09fb).
```

# JHipster updates 

## JHipster 6

JHipster 6 was released last month with all its major components upgraded to their latest release (Spring Boot, Angular, React, JUnit 5, etc).
Other important changes that comes with version 6:
- JDK 11 with JDK 8 compatibility
- Default packaging changed to Jar (make Jar not War)
- Bootswatch theme selection

## Cloud support

> Overall, the cloud support is improved with the latest Spring Cloud release train: Greenwhich. Helm is now supported with its own sub-generator and the Kubernetes/Istio support was also improved.

Reviewer: ```suggestion
Overall, the cloud support was improved with the latest Spring Cloud release train: Greenwhich. Helm is now supported with its own sub-generator and the Kubernetes/Istio support was also improved.
```

## Vue.js

> [Vue.js](https://vuejs.org/) is the new trending framework in the Javascript front-end world, and its simplicity lets you build web applications pretty fast. I really recommend trying the official [Vue.js blueprint](https://github.com/jhipster/jhipster-vuejs), here a [blog post](https://blog.ippon.tech/creating-a-modern-web-app-using-vuejs-and-spring-boot-with-jhipster/) that explains how Vue.js is different than Angular and React. The JHipster core team is also discussing on merging the blueprint in the main project which will simplify its usage.

Reviewer: ```suggestion
[Vue.js](https://vuejs.org/) is the new trending framework in the Javascript front-end world, and its simplicity lets you build web applications lightning fast. I highly recommend trying the official [Vue.js blueprint](https://github.com/jhipster/jhipster-vuejs) and here is a [blog post](https://blog.ippon.tech/creating-a-modern-web-app-using-vuejs-and-spring-boot-with-jhipster/) that explains the differences between Vue.js, Angular, and React. The JHipster core team is also discussing merging the blueprint in the main project, which will simplify its usage.
```

## Project metrics

> With 100,000 installations per month and more than 2 million in total, JHipster popularity is having a steady grow. The [main website](https://www.jhipster.tech/) gets 90,000 unique visitors per month and the project on [Github](https://github.com/jhipster/generator-jhipster) has more than 14,000 stargazers!

Reviewer: ```suggestion
With 100,000 installations per month and more than 2 million in total, JHipster popularity is steadily growing. The [main website](https://www.jhipster.tech/) gets 90,000 unique visitors per month and the project on [Github](https://github.com/jhipster/generator-jhipster) has more than 14,000 stargazers!
```

## Miscellaneous

> If you're new to the JHipster world, you can follow a serie of [free tutorials here](https://github.com/jhipster/jhipster-guides). All you need is a Google account and you will be able to play with JHipster using Google Cloud Shell.

Reviewer: ```suggestion
If you're new to the JHipster world, you can follow this series of [free tutorials](https://github.com/jhipster/jhipster-guides). All you need is a Google account and you will be able to play with JHipster using Google Cloud Shell.
```

> In case you did not know, the whole JHipster's artwork was updated with a new logo and a whole JHipster family. More details can be found on the [official page](https://www.jhipster.tech/artwork/).

Reviewer: ```suggestion
In case you did not know, JHipster's artwork was updated with a new logo and a whole new JHipster family. More details can be found on the [official page](https://www.jhipster.tech/artwork/).
```

# Roadmap

## JDL centrict configuration

![JDL](https://github.com/falydoor/blog-usa/blob/jhipster-conf-2/images/2019/07/jdl.png)

## Prettier for Java

> [Prettier](https://prettier.io/) is an opinionated code formatter and it helps the team having an unified code formatter for the whole project. It is used by JHipster for the javascript/typescript files but Prettier doesn't support Java right now. So instead of waiting, the team decided to [create the plugin](https://github.com/jhipster/prettier-java)!

Reviewer: ```suggestion
[Prettier](https://prettier.io/) is an opinionated code formatter and it helps the team by having a unified formatter for the whole project. It is used by JHipster for the javascript/typescript files but Prettier doesn't support Java right now. So, instead of waiting, the team decided to [create the plugin themselves](https://github.com/jhipster/prettier-java)!
```

# Videos

All the talks of the JHipster Conf 2 can be viewed on the [youtube playlist](https://www.youtube.com/playlist?list=PL6IFaLdAcgE3aSgmRyhi6eULdlrwjA5sm).

> To resume, JHipster Conf 2019 with its English and French track was a huge success. This success was made possible by having high quality and diversity talks, many thanks to our rockstars speakers! And also thanks to our sponsors: [Ippon Technologies](https://www.ippon.tech/), [Société Générale](https://www.societegenerale.com/), [Microsoft](https://www.microsoft.com/), [Okta](https://www.okta.com/) and [Heroku](https://www.heroku.com/).

Reviewer: ```suggestion
JHipster Conf 2019 with its English and French track was a huge success. This success was made possible by having high quality and diverse talks, many thanks to our rockstar speakers! Also thanks to our sponsors: [Ippon Technologies](https://www.ippon.tech/), [Société Générale](https://www.societegenerale.com/), [Microsoft](https://www.microsoft.com/), [Okta](https://www.okta.com/) and [Heroku](https://www.heroku.com/).
```
